src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -48,7 +52,4 @@ public default void setLeftVelocitySetpoint(double velocityRadiansPerSecond) {}
 
   /** Sets motor closed loop velocity setpoint for right flywheel. */
   public default void setRightVelocitySetpoint(double velocityRadiansPerSecond) {}
```

Comment:
It is probably worth making this function setRightGoalVelocity, as in a motion profile, the setpoint is determined by the profile constraints.